CHANGELOG.md

```diff
@@ -1,4 +1,29 @@
 ## 3.20.0 (Unreleased)
+
+* **New Resource:** `google_artifact_registry_repository` (Beta only) ([#1981](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1981))
```

Comment:
Beta only messages aren't strictly needed, but could be helpful so fine either way. 🤷‍♀️